Makefile

```diff
@@ -59,17 +59,25 @@ non_windows_platforms := darwin_amd64 darwin_arm64 linux_amd64 linux_arm64
 # TODO: arm64 on Windows https://github.com/envoyproxy/envoy/issues/17572
 windows_platforms := windows_amd64
 
-$(build_dir)/func-e_%/func-e: main.go
-	$(call go-build, $@, $^)
+# TODO: To have expression that understand globbing for any depth.
+gocodes := [!_test].go
```

Comment:
since our code is always in the internal subdir (except main.go) can we tighten this?